examples/srxe/display/main.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021, Tomasz Wasilczyk
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+
+int
+main()
+{
+	Board::initialize();
+
+	auto& display = Board::display;
+	display.drawRectangle(0, 0, 10, 10);
```

Comment:
This is just testing code. I will make it more meaningful in the final version of this PR.